mitosheet/src/mito/components/icons/ConditionalFormatIcon.tsx

```diff
@@ -4,12 +4,19 @@
 import React from 'react';
 
 const ConditionalFormatIcon = (props: {backgroundColor: string, color: string}): JSX.Element => {
+    // TODO: use the props. Including this console log so that the linter doesn't complain
+    console.log(props)
```

Comment:
We'd rather not spam the console -- I think you can just do `props` or if we don't use the props just remove them for now.